### PR TITLE
feat: add certifications to tables

### PR DESCRIPTION
**Pull request by etr2460**
### SUMMARY
Adds certification details to tables in the new CRUD view and allows editing in both the old and new dataset editors

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Screen Shot 2020-10-27 at 3 51 11 PM
Screen Shot 2020-10-27 at 3 50 55 PM
Screen Shot 2020-10-27 at 3 50 43 PM

### TEST PLAN
Add extra data to a table, see the certification details
Also test invalid data in extra and see no JS exceptions

### ADDITIONAL INFORMATION

- [x] Has associated issue: #10591 
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

to: @nytai @ktmud @john-bodley

### CODE REVIEW

**Comment by ktmud** [comment]
Should we add the verification icon to be after the table name? The alignment looks kind of weird right now. Imaging you have two tables starting with the same prefix and only one of them is verified:

```
booking_value_per_nights_by_country
[V] booking_value_per_nights_by_home_type
```

```
booking_value_per_nights_by_country
booking_value_per_nights_by_home_type [V]
```

**Comment by etr2460** [comment]
Design preference has typically been for the verification icon before the entity name. This is used in metrics in superset, and also aligns to specs from @kenchendesign

**Comment by rusackas** [comment]
another rebase _should_ fix the broken E2E test.

File: superset-frontend/src/datasource/DatasourceEditor.jsx [hunk 1]
```diff
@@ -516,6 +516,25 @@ class DatasourceEditor extends React.PureComponent {
...
           />
         )}
+        {this.state.isSqla && (
+          <Field
+            fieldKey="extra"
+            label={t('Extra')}
+            description={t(
+              'Extra data to specify table metadata. Currently supports ' +
+                'certification data of the format: `{ "certification": { "certified_by": ' +
+                '"Taylor Swift", "details": "This table is the source of truth." ' +
```
Comment (ktmud) [inline superset-frontend/src/datasource/DatasourceEditor.jsx:526]: Lol, I like how we codified `Taylor Swift` to the actual user interface. She's certainly not a controversial figure, but should we use a more "business friendly" example? E.g. `"certified_by": "Data Platform Team"`

File: superset/connectors/sqla/models.py [hunk 2]
```diff
@@ -492,6 +492,7 @@ class SqlaTable(  # pylint: disable=too-many-public-methods,too-many-instance-at
     sql = Column(Text)
     is_sqllab_view = Column(Boolean, default=False)
     template_params = Column(Text)
+    extra = Column(Text)
```
Comment (ktmud) [inline superset/connectors/sqla/models.py:495]: We don't need a db migration for this? If so, maybe add a link in this PR to the migration script that has added this column?

**Review by ktmud — COMMENTED** [review]
LGTM, 2 small nits

**Review by ktmud — APPROVED** [review]

File: superset-frontend/src/datasource/DatasourceEditor.jsx (hunk 1 above)
Comment (mistercrunch) [reply superset-frontend/src/datasource/DatasourceEditor.jsx]: +1 on Taylor Swift
Screen Shot 2020-10-31 at 1 44 50 PM

Or Batman. Batman may work too.

Comment (mistercrunch) [reply superset-frontend/src/datasource/DatasourceEditor.jsx]: Actually, let's draw the line on examples (where random/funny is ok) and keep tooltips / descriptions more businessy

**Pull request merged**